plugin/src/main/java/com/denizenscript/denizen/events/block/BrewingStartsScriptEvent.java

```diff
@@ -0,0 +1,70 @@
+package com.denizenscript.denizen.events.block;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BrewingStartEvent;
+
+public class BrewingStartsScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // brewing stand starts brew

```

Comment:
That event line sounds a bit weird imo, and also doesn't match what's in `registerCouldMatcher` - probably `brewing stand starts brewing`? or maybe `brewing starts`?